### PR TITLE
EventPreview - Connect to SDK

### PR DESCRIPTION
**Pull request by xitman78**
Added wrapper component that fetches event and provides data to the inner view component.
Proper documentation will be added within a new corresponding issue.

### CODE REVIEW

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-176.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-176.gearboxjs.preview.cogniteapp.com

File: src/components/EventPreview/EventPreview.tsx [hunk 1]
```diff
@@ -1,87 +1,76 @@
...
 `;
 
-export const defaultStrings: PureObject = {
-  noDescription: 'No description',
-  start: 'Start',
-  end: 'End',
-  details: 'Explore event details',
-  metadataSummary: 'Contains {{count}} additional pieces of data',
-};
+export const LoadingSpinner: React.SFC = () => (
```
Comment (follan) [inline src/components/EventPreview/EventPreview.tsx]: Does this need to be exported?

Comment (xitman78) [reply src/components/EventPreview/EventPreview.tsx]: No, it doesn't! Just fixed.

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-176.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-176.gearboxjs.preview.cogniteapp.com

**Pull request merged**